Update documentation for pagination (#1435)
google/osv.dev: docs/api/post-v1-query.md

  ]
}

```

## Pagination

> For queries that result in a large number of vulnerabilities, the response will be paginated. As long as "next_page_token" is given in the response, there are more results to return. 

Reviewer: ```suggestion
For queries that result in a large number of vulnerabilities, the response will be paginated. In this case, the "next_page_token" is given in the response, indicating that there are more results to return. You will need to run additional queries using the `page_token` to see the remaining results, repeating queries until the `next_page_token` is no longer included in the response. 
```

> The response will be in this form:

Reviewer: ```suggestion
A response indicating pagination will be in this form:
```

```json
{
  "vulns": [
    ...
  ],
  "next_page_token": "a base64 string here"
}
```

> To get the next page of results, your next request should be in the form:

Reviewer: ```suggestion
To get the next page of results, your next request must include page_token:
```
